src/mbed_tools/cli/configure.py

```diff
@@ -66,6 +53,6 @@ def configure(output_directory: Any, toolchain: str, mbed_target: str, program_p
     target_build_attributes = get_target_by_name(mbed_target, program.mbed_os.targets_json_file)
     config = assemble_config(Source.from_target(target_build_attributes), program.root, program.files.app_config_file)
     cmake_file_contents = generate_mbed_config_cmake_file(mbed_target, target_build_attributes, config, toolchain)
-    output_directory = pathlib.Path(output_directory, ".mbedbuild")
+    output_directory = program.root / ".mbedbuild"
```

Comment:
I'm unfamiliar with this syntax. You can really divide by a string and append to the path? If not, probably we should use some sort of path joining function.

Reply:
`pathlib.Path` overloads the `/` operator to perform the same function as `join`.

Reply:
Thanks for explaining. I figured it'd be some operator overloading magic given the unit tests pass and this code is included in the tests.